### PR TITLE
Websockets create new meeting

### PR DESCRIPTION
**Pull request by MirkoTV**
- Implementing web sockets for create new meeting functionallity
- Using isotope instead of masonry for sorting the cards because isotope gives you the option of sorting by a parameter
- Using Jsrender to create every foodMeeting card

### CODE REVIEW

File: src/main/java/org/jala/efeeder/foodmeeting/CreateFoodMeetingCommand.java [hunk 1]
```diff
@@ -5,60 +5,78 @@
...
  */
 @Command
 public class CreateFoodMeetingCommand implements CommandUnit {
 
-	private static final String INSERT_FOOD_MEETING_SQL = "insert into food_meeting(name,image_link, status, event_date, created_at) "
-			+ "values(?, ?, ?, ?, ?)";
-	private static final String FOOD_MEETING_STATUS = FoodMeetingStatus.Voting.name();
+    private static final String INSERT_FOOD_MEETING_SQL = "insert into food_meeting(name,image_link, status, event_date, created_at) "
+            + "values(?, ?, ?, ?, ?)";
+	private static final String creatMeetingRoomId = "creatMeetingRoomId";
```
Comment (ziur) [inline src/main/java/org/jala/efeeder/foodmeeting/CreateFoodMeetingCommand.java]: Typo in the name, and a constant should be in upper case

File: src/main/java/org/jala/efeeder/foodmeeting/CreateFoodMeetingCommand.java [hunk 2]
```diff
@@ -5,60 +5,78 @@
...
+        Out out = new DefaultOut();
+        
+        String roomId = context.getMessageContext().getRoom().toString();
+        PreparedStatement stm = context.getConnection()
+                                        .prepareStatement(INSERT_FOOD_MEETING_SQL, RETURN_GENERATED_KEYS);
+        
+        List<MessageEvent> messages = context.getMessageContext().getEvents();        
+        CreateFoodMeetingEvent createMeetingEvent = (CreateFoodMeetingEvent)messages.get(0).getEvent();
+        
+		Timestamp eventDate = new Timestamp(Long.parseLong(createMeetingEvent.getEventDate().toString()));
```
Comment (ziur) [inline src/main/java/org/jala/efeeder/foodmeeting/CreateFoodMeetingCommand.java:42]: Align code

File: src/main/java/org/jala/efeeder/foodmeeting/CreateFoodMeetingCommand.java [hunk 3]
```diff
@@ -5,60 +5,78 @@
...
+				.setName(foodMeeting.getName())
+				.setImageLink(foodMeeting.getImageLink())
+				.setStatus(foodMeeting.getStatus())
+				.setEventDate(foodMeeting.getEventDate().getTime())
+				.setWidth(foodMeeting.getWidth())
+				.build()
+			)
+			.build()
+		);
+		MessageContext messageContext = MessageContext.newBuilder()
```
Comment (ziur) [inline src/main/java/org/jala/efeeder/foodmeeting/CreateFoodMeetingCommand.java]: Please, fix identation in these lines

File: src/main/java/org/jala/efeeder/wheel/WheeldecideCommand.java [hunk 4]
```diff
@@ -75,8 +75,9 @@ public Out execute(In context) throws Exception {
     private static int insertNewBuyer(In context, int choseUserId) throws Exception{
         PreparedStatement stm = context.getConnection()
                                         .prepareStatement("insert into buyer(id_food_meeting, id_user) values(?, ?)");
-
-        stm.setInt(1, context.getMessageContext().getRoom());
+		
+		String roomId = context.getMessageContext().getRoom().toString();
```
Comment (ziur) [inline src/main/java/org/jala/efeeder/wheel/WheeldecideCommand.java:77]: Fix identation

File: src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java [hunk 5]
```diff
@@ -12,42 +12,49 @@
...
+		this.id = id;
+		this.name = name;
+		this.createdAt = new Timestamp(System.currentTimeMillis());
+		this.eventDate = eventDate;
+		this.imageLink = imageLink;
+		this.status = DEFAULT_FOOD_MEETING_STATUS;
+	}
+
+	public int getWidth()
+	{
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java]: please, follow Java style code

File: src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java [hunk 6]
```diff
@@ -12,42 +12,49 @@
...
+		int width = 500;
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java]: In any case, this value will be changed if you use the 'width' variable to store this value, it seems 500 is the default value to be returned

File: src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java [hunk 7]
```diff
@@ -12,42 +12,49 @@
...
+
+		LocalDate today = new LocalDate();
+		LocalDate endEventDate = new LocalDate(eventDate);
+
+		int days = Days.daysBetween(today, endEventDate).getDays();
+
+		width = (50 * days) < 300 ?
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java]: I think this operation should be a function

File: src/main/java/org/jala/efeeder/wheel/WheeldecideCommand.java [hunk 8]
```diff
@@ -27,62 +27,63 @@
...
+		List<User> users = new ArrayList<>();
+		while (resultSet.next()) {
+			int idUser = resultSet.getInt(1);
+			String nameUser = resultSet.getString(2);
+			users.add(new User(idUser, nameUser));
+		}
+		Collections.shuffle(users);
+		choseIndex = getRandomIndexPerson(users.size());
+
+		//insertNewBuyer(context, users.get(choseIndex).getId());
```
Comment (Ramir0) [inline src/main/java/org/jala/efeeder/wheel/WheeldecideCommand.java]: remove this line

**Review by Ramir0 — APPROVED** [review]

**Pull request merged**